### PR TITLE
zope.event 5.0

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**Destination channel:** main

### Links

- [PKG-3684](https://anaconda.atlassian.net/browse/PKG-3684) 
- [Upstream repository](https://github.com/zopefoundation/zope.event)
- Changelog: https://github.com/zopefoundation/zope.event/blob/5.0/CHANGES.rst
- License: https://github.com/zopefoundation/zope.event/blob/5.0/LICENSE.txt
- Requirements: https://github.com/zopefoundation/zope.event/blob/5.0/setup.py

### Explanation of changes:

- Add the feedstock's license
- Remove redundant jinja variables
- Update `script`
- Add missing `pip` and `wheel` to `host`
- Add `pip check`
- Add `pytest`
- Update `home`, doc & dev url
- Update `license` as `ZPL-2.1`
- Update `summary`

### Notes:

- `gevent 23.9.1` requires zope.event for py312 https://github.com/AnacondaRecipes/gevent-feedstock/pull/5

[PKG-3684]: https://anaconda.atlassian.net/browse/PKG-3684?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+BSD 3-clause license
+Copyright (c) 2015-2020, conda-forge contributors
```
Comment (JeanChristopheMorinPerso) [inline LICENSE.txt]: We should use the Anaconda license here I think.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,48 +1,56 @@
...
 
 requirements:
   host:
     - python
+    - pip
     - setuptools
+    - wheel
   run:
     - python
+    # https://github.com/zopefoundation/zope.event/blob/5.0/setup.py#L69
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]:
```suggestion
    # https://github.com/zopefoundation/zope.event/blob/5.0/setup.py#L70
```

**Review by lorepirri — APPROVED** [review]

File: LICENSE.txt (hunk 1 above)
Comment (skupr-anaconda) [reply LICENSE.txt]: Fixed

File: recipe/meta.yaml (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Fixed

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**